modules/sugar/src/main/ts/ephox/sugar/api/search/Traverse.ts

```diff
@@ -8,7 +8,7 @@ import * as SugarNode from '../node/SugarNode';
  * The document associated with the current element
  * NOTE: this will throw if the owner is null.
  */
-const owner = (element: SugarElement<Node>) => SugarElement.fromDom(element.dom.ownerDocument);
+const owner = (element: SugarElement<Node>) => SugarElement.fromDom(element.dom.ownerDocument as Document);
```

Comment:
We maybe should consider changing this and `defaultView` below to an `Optional` instead of making it throw an error. That's outside the scope of this PR though.